tests/test-webpack-stats.json

```diff
@@ -0,0 +1,28 @@
+{
+    "chunks": {
+        "main": [
+            {
+                "name": "static/js/main.9a35969a.js",
+                "path": "/Users/jbaldwin/work/appsembler/devstacks/gfig3/src/figures/figures/static/figures/static/js/main.9a35969a.js",
```

Comment:
This is an interesting one.

BundleTracker auto-generates this file, which is needed by Django Webpack loader to locate assets. As for the "path" variable, it seems that webpack loader simply ignores it. I could play around  by removing the "path" from it and see if it breaks webpack loader. 

I can look into this some more because even though it appears to be an unused artifact, I don't like it either.